src/GDGUkraine/model.py

```diff
@@ -151,6 +151,7 @@ def __init__(self, **kwargs):
     date = Column(Date)
     closereg = Column(Date)
     fields = deferred(Column(JSONEncodedDict(512)))
+    hidden = deferred(Column(JSONEncodedDict(512)))
```

Comment:
@anxolerd 
Every time you make change to ORM model you **have to** create appropriate DB migrations.